File: src/keyboardRow.java
import java.util.ArrayList;

public class keyboardRow {
	public String[] findWords(String[] words) {
		ArrayList<String> out = new ArrayList<String>();
		for(String word : words) {
			if(word.toLowerCase().matches("[qwertyuiop]*|[asdfghjkl]*|[zxcvbnm]*")) {
				out.add(word);
			}
		}
		return out.toArray(new String[out.size()]);
	}
	
	public static void main(String[] args) {
		keyboardRow x = new keyboardRow();
		System.out.println(x.findWords(new String[] {"Hello", "Alaska", "Dad", "Peace"}));
	}
}

File: JS/keyboardRow.js
/**
 * @param {string[]} words
 * @return {string[]}
 */
var findWords = function(words) {
    return words.filter(x => x.match(/[qwertyuiop]+$|[asdfghjkl]+$|[zxcvbnm]+$/gi).join("") == x);
};

console.log(findWords(["Hello", "Alaska", "Dad", "Peace"]));